scripts/cnv_3_gistic_style.py

```diff
@@ -121,18 +120,17 @@ def mt_adjust_cn(obj):
 
     x = 1
     m = 50
-    with concurrent.futures.ThreadPoolExecutor(config_data["threads"]) as executor:
-        results = {
-            executor.submit(mt_adjust_cn, bs_id): bs_id for bs_id in bs_cbio_dict
-        }
-        for result in concurrent.futures.as_completed(results):
-            if result.result()[0] == 1:
-                "Had trouble processing object " + result.result([1] + "\n")
-                sys.exit(1)
-            if x % m == 0:
-                sys.stderr.write("Processed " + str(x) + " samples\n")
-                sys.stderr.flush()
-            x += 1
+
+    for bs_id in bs_cbio_dict:
+        result = mt_adjust_cn(bs_id)
```

Comment:
When your function returns multiple items you can declare them like this. Makes this a bit easier to understand.
```suggestion
        exit_code, object = mt_adjust_cn(bs_id)
```

Reply:
I'll make the change, but also would need to change downstream references. Not hard, but doing this alone would break it

Reply:
I just put it here as an example. What you have works. Obviously the code needs a much bigger cleanup than this change alone.